### PR TITLE
Used derived element name for closing tag as well

### PR DESCRIPTION
**Pull request by khankuan**
Opening tag was fixed in #46 but we need to apply the logic to closing tags as well.

Cheers!

### CODE REVIEW

**Comment by khankuan** [comment]
@cpojer ping :)

**Comment by cpojer** [comment]
hey! Seems like it isn't passing on travis. Would you mind trying to make that work? I think it is because node 4 doesn't infer function names properly.

**Comment by khankuan** [comment]
Fixed!

File: __tests__/test.js [hunk 1]
```diff
@@ -526,6 +526,16 @@ describe('prettyFormat()', () => {
       );
     });
 
+    it('supports a single element with custom React elements with a child', () => {
+      function Cat() {
```
Comment (thejameskyle) [inline __tests__/test.js]: What is the point of the function here?

Comment (khankuan) [inline __tests__/test.js]: Sharp. This test is pretty wrong, fixing it shortly.

Comment (cpojer) [inline __tests__/test.js]: I removed it.

**Pull request merged**